### PR TITLE
Update dex page with the cluster CA addition

### PR DESCRIPTION
**Pull request by pipo02mix**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/advanced/configure-dex-in-your-cluster/index.md [hunk 1]
```diff
@@ -216,7 +216,7 @@ data:
 
 __Warning__: With `oidc` connector you might need to add `getUserInfo` in the connector configuration to force a second call to the identity provider in order to get groups. This is required for example by Okta. More info on this can be found in [dexipd/dex#1065](https://github.com/dexidp/dex/issues/1065).
 
-__Note__: In the above snippet you have to replace the `<CLUSTERID>` variable and add the Kubernetes Certificate Authority to ensure Dex can trust the API endpoint. Finally you have to use a connector. Here we show examples for Keycloak, Active Directory, and GitHub.
+__Note__: In the above snippet you have to replace the `<CLUSTERID>` variable and select a connector. Here we show examples for Keycloak, Active Directory, and GitHub.
```
Comment (pipo02mix) [inline src/content/advanced/configure-dex-in-your-cluster/index.md:219]: We dont need to add the CA anymore

Comment (marians) [reply src/content/advanced/configure-dex-in-your-cluster/index.md:219]: Why? Is it not needed any more for k8s-dex-authenticator? See https://docs.giantswarm.io/ui-api/management-api/authentication/login-utility-results.png

**Comment by marians** [comment]
Is it a coincidence? In Rainbow we discussed the topic this morning and created https://github.com/giantswarm/roadmap/issues/762

My understanding was that the CA pem field is required for k8s-dex-authenticator and should be explained better.

File: src/content/advanced/configure-dex-in-your-cluster/index.md (hunk 1 above)
Comment (pipo02mix) [reply src/content/advanced/configure-dex-in-your-cluster/index.md:219]: it is taken from cluster configmap values

Comment (pipo02mix) [reply src/content/advanced/configure-dex-in-your-cluster/index.md:219]: so customer does not need to provide it when they use `customer` connector

**Comment by pipo02mix** [comment]
We have included in the cluster configmap values so they don't have to provide it, same as base domain, we are defaulting it, we can always explain the properties if we want

File: src/content/advanced/configure-dex-in-your-cluster/index.md (hunk 1 above)
Comment (marians) [reply src/content/advanced/configure-dex-in-your-cluster/index.md:219]: Cool, thank you!

**Review by marians — APPROVED** [review]

**Pull request merged**